src/KeyVault/KeyVault/help/Undo-AzKeyVaultSecretRemoval.md

```diff
@@ -162,6 +162,6 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Remove-AzKeyVaultSecret](./Remove-AzKeyVaultSecret.md)
 
-[Add-AzKeyVaultSecret](./Add-AzKeyVaultSecret.md)
```

Comment:
Is this removed by mistake?

Reply:
The removal was intentional. The link is invalid due to no PowerShell command by that name existing in any of the Az PowerShell modules.